### PR TITLE
docs: setup with service account docs

### PR DESCRIPTION
**Pull request by stebenz**

### CODE REVIEW

**Comment by mffap** [comment]
@eliobischof @hifabienne can you pls provide me with an ETA when this PR will be reviewed?

**Comment by eliobischof** [comment]
> @eliobischof @hifabienne can you pls provide me with an ETA when this PR will be reviewed?

I'll review soon, after dinner 🙂

**Comment by mffap** [comment]
> > @eliobischof @hifabienne can you pls provide me with an ETA when this PR will be reviewed?
> 
> I'll review soon, after dinner 🙂

😀 Ok, not *that* urgent. Thanks!

**Comment by eliobischof** [comment]
To me, it feels very odd to duplicate all platform installation instructions like this. I think, we should keep them as simple and maintainable as possible.

I'd rather just describe what options a user has and how key generation works in a separate page beneath self-hosted.  This page would then also be linkable from the terraform page, for example.

**Review by eliobischof — CHANGES_REQUESTED** [review]
https://github.com/zitadel/zitadel/pull/4877#issuecomment-1353490466

File: docs/docs/guides/deploy/kubernetes.mdx [hunk 1]
```diff
@@ -31,6 +33,32 @@ helm install --namespace zitadel --create-namespace my-zitadel zitadel/zitadel \
   --set cockroachdb.statefulset.replicas=1
 ```
 
+#### Install an insecure zitadel release that works with localhost with a service account
```
Comment (eliobischof) [inline docs/docs/guides/deploy/kubernetes.mdx:36]: can you move the DefaultUser component above this line, please?

**Review by eliobischof — CHANGES_REQUESTED** [review]

**Review by eliobischof — DISMISSED** [review]
Created a follow-up issue in order to resolve the disaggreement #4892

**Review by eliobischof — APPROVED** [review]

**Pull request merged**